### PR TITLE
add README-CN.md 

### PR DESCRIPTION
**Pull request by yunshuipiao**
for Chinese students

### CODE REVIEW

**Comment by keon** [comment]
my only concern is that if it can keep up to date with the main README.md 
I agree that having more language will improve the accessibility though.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+### English | [简体中文](https://github.com/yunshuipiao/algorithms/blob/master/README_CN.md)
```
Comment (keon) [inline README.md]: I think making it smaller would be better. (normal font size)
example: https://github.com/kdn251/interviews

Comment (yunshuipiao) [reply README.md]: @keon thank you .I will fix it .

Comment (yunshuipiao) [reply README.md]: @keon fix it . I will keep up to date with main README.md and the Chinese part as far as I can.

**Pull request merged**